msal/src/main/java/com/microsoft/identity/client/internal/CommandParametersAdapter.java

```diff
@@ -486,4 +492,28 @@ public static GenerateShrCommandParameters createGenerateShrCommandParameters(
                 .popParameters(popParameters)
                 .build();
     }
+
+    /**
+     * Adds additional query string parameters to original list.
+     * @param queryStringParameters extra query string parameters inputted.
+     * @param configuration configuration created from MSAL config file.
+     * @return combined list of query string parameters.
+     */
+    @Nullable
+    private static List<Map.Entry<String, String>> getExtraQueryStringParameters(
```

Comment:
nit: appendWebAuthNToExtraQueryParameters?

Reply:
what about appendToExtraQueryParametersIfWebAuthnCapable? Cause we only append it if the property is set.